Download.Mvc/Client/components/FolderEntries.tsx

```diff
@@ -0,0 +1,57 @@
+﻿import * as React from 'react';
+import { FolderEntry } from './FolderEntry';
+import { FolderParent } from './FolderParent';
+import { listStyle, iconStyle } from './FolderView';
+import { FilePreview } from './FilePreview';
+
+export class FolderEntries extends React.Component<any, any>{
+    constructor(props) {
+        super(props);
+    }
+
+    getReadmeID = (id) => {
+        this.props.getReadmeID(id);
+    }
+
+    getLicenseID = (id) => {
+        this.props.getLicenseID(id);
+    }
+
+    render() {
+        let entryList = this.props.data.item_collection.entries.map((entry, index) =>
+            <FolderEntry key={index} entry={entry} updateReadmeID={this.getReadmeID} updateLicenseID={this.getLicenseID} licenseID={this.props.licenseID}/>
```

Comment:
@laholstege you could always just call this.props.getReadmeID in here directly and save yourself the trouble of making local function, they don't actually do anything.